exporter/jaeger/jaegerthrifthttpexporter/exporter_test.go

```diff
@@ -57,21 +49,39 @@ func TestNew(t *testing.T) {
 			},
 		},
 	}
+
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got, err := New(tt.args.config, tt.args.httpAddress, tt.args.headers, tt.args.timeout)
-			if (err != nil) != tt.wantErr {
-				t.Errorf("New() error = %v, wantErr %v", err, tt.wantErr)
-				return
-			}
-
-			if got == nil {
-				return
-			}
+			assert.NoError(t, err, "nil config")
+			assert.NotNil(t, got)
 
 			// This is expected to fail.
 			err = got.ConsumeTraceData(context.Background(), consumerdata.TraceData{})
 			assert.Error(t, err)
 		})
 	}
 }
+
+func TestNewFails(t *testing.T) {
+	tests := []struct {
```

Comment:
Since there is only one test here the value of having an array of test definitions is unclear here.

Reply:
I assumed that other cases could be added here in the future. But you're right, for now there is only one. So I fixed this test.

Reply:
Let's use the principle of YAGNI here. :-)
We generally aim to have minimum amount of code that serves the purpose.